runners/spark/src/main/java/org/apache/beam/runners/spark/translation/TransformTranslator.java

```diff
@@ -776,7 +777,7 @@ public Boolean call(Tuple2<TupleTag<V>, WindowedValue<?>> input) {
     EVALUATORS.put(HadoopIO.Write.Bound.class, writeHadoop());
     EVALUATORS.put(ParDo.Bound.class, parDo());
     EVALUATORS.put(ParDo.BoundMulti.class, multiDo());
-    EVALUATORS.put(GroupByKey.GroupByKeyOnly.class, gbk());
+    EVALUATORS.put(GroupByKeyOnly.class, gbk());
```

Comment:
Given that this also removes the default GroupByKey expansion, will this work without adding a runner-specific expansion?